pkgs/top-level/all-packages.nix

```diff
@@ -16101,7 +16101,8 @@ with pkgs;
   };
   swiPrologWithGui = swiProlog.override { withGui = true; };
 
-  tbb = callPackage ../development/libraries/tbb { };
+  inherit (callPackages ../development/libraries/tbb { })
+    tbb tbb_2020_3 tbb_2021_8;
```

Comment:
One usually keeps the default version in `default.nix` and puts a held back version in versioned file, like:

```nix
tbb = callPackage ../development/libraries/tbb { };
tbb_2020_03 = callPackage ../development/libraries/tbb/2020.03.nix { };
```

Having a `default.nix` not to return a package is a bit unusual.

Reply:
Updating the default (tbb) to tbb_2021_8 would break half of the packages using it (https://github.com/NixOS/nixpkgs/pull/214762#issuecomment-1418286637), so it is out-of-scope of this PR!

But i will make the latest version default.nix.

Reply:
Aha. If a new version breaks a lot, then making a previous version as a `default.nix` sounds fine as well. Then default change could be done separately.